rolling-deploy.sh

```diff
@@ -136,9 +136,16 @@ done
 
 cf start "${NEW_APP_NAME}"
 
+# do any additional work such as testing, bindings, etc
+# shellcheck disable=SC1090
+[[ -f "${ARTIFACT_PATH}/post-deploy.sh" ]] && . "${ARTIFACT_PATH}/post-deploy.sh"
```

Comment:
this allows any var defined to be referenced in a hook script, no args needed. IF file doesn't exist, no-op